# timestamp of the state
time stamp

# States of the instances of the three tray types
TrayState[] tray_states
